src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1516,8 +1524,11 @@ ucs_status_t uct_dc_mlx5_ep_check_fc(uct_dc_mlx5_iface_t *iface,
     }
 
     now = ucs_get_time();
-    it  = kh_put(uct_dc_mlx5_fc_hash, &iface->tx.fc_hash, (uint64_t)ep,
-                 &ret);
+    if (kh_size(&iface->tx.fc_hash) == 0) {
```

Comment:
but we should initialize it here, otherwise - going over all EPs in `uct_dc_mlx5_ep_fc_hard_req_progress` will occur immediately, but we don't want it

Reply:
> does it make sens to move it before line 1560 and check cb_id == NULL_ID?

done

> also the function name is a bit awkward

renamed